applications/connectivity_bridge/src/main.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+
+#include <event_manager.h>
+
+#define MODULE main
+#include "module_state_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE);
+
+/* Overriding weak function to set iSerial runtime. */
+u8_t *usb_update_sn_string_descriptor(void)
+{
+	static u8_t buf[] = "PCA20035_12PLACEHLDRS";
```

Comment:
I get that returning an address that points at buf is safe because it's a static variable, but it looks really scetchy, would you mind placing buf in the file-level scope so that it is even more clear (to the reader, not the compiler) that this is not a stack address?